ravindu14/SliitSpmProject
# Hotfix task#1156

how to test
1. npm install
2. ng serve -o
3. navigate to "http://localhost:4200/tables"
4. check interface input alignment fixed
